src/ui.toit

```diff
@@ -340,19 +448,83 @@ class Ui:
         if message:
           printer_.emit --kind=kind "$text.call"
 
+  /** Variant of $(emit --kind [--structured] [--text]) using the $DEBUG kind. */
+  emit --debug/bool [--structured] [--text]:
```

Comment:
Maybe all of these should have `-> none`.

Reply:
done.